spec/acceptance/rest/auth_spec.rb

```diff
@@ -1365,5 +1365,18 @@ def level=(new_level)
         expect(response).to be_a(Ably::Models::TokenDetails)
       end
     end
+
+    context 'when using JWT' do
+      it 'client pulls non nil stats when authenticating with a JWT encoded token' do
+        require 'jwt'
```

Comment:
It's quite unusual to require this within a block.  I know it works, but it's not best practice.  Can we include all dependencies at the start of this file please?